config.cfg

```diff
@@ -1,11 +1,14 @@
+########################################################################
```

Comment:
# General
There are many parameters and tasks in this file that are not explained. Seeing this for the first time is daunting.